mico-core/src/main/java/io/github/ust/mico/core/model/MicoServiceDeploymentInfo.java

```diff
@@ -155,92 +163,89 @@
-                "Defaults to DEFAULT.")
+            @ExtensionProperty(name = "description", value = "Indicates whether and when to pull the image.")
         }
     )})
-    private ImagePullPolicy imagePullPolicy = ImagePullPolicy.DEFAULT;
+    @JsonSetter(nulls = Nulls.SKIP)
+    private ImagePullPolicy imagePullPolicy = ImagePullPolicy.ALWAYS;
 
     /**
      * Restart policy for all containers.
-     * Defaults to RestartPolicy#ALWAYS.
+     * Default restart policy is {@link RestartPolicy#ALWAYS}.
+     * {@code null} is ignored.
      */
     @ApiModelProperty(extensions = {@Extension(
         name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
         properties = {
             @ExtensionProperty(name = "title", value = "Restart Policy"),
+            @ExtensionProperty(name = "default", value = "ALWAYS"),
             @ExtensionProperty(name = "x-order", value = "70"),
-            @ExtensionProperty(name = "description", value = "Restart policy for all containers.\n" +
-                " Defaults to ALWAYS.")
+            @ExtensionProperty(name = "description", value = "Restart policy for all containers.")
         }
     )})
-    private RestartPolicy restartPolicy = RestartPolicy.DEFAULT;
-    
-    
+    @JsonSetter(nulls = Nulls.SKIP)
+    private RestartPolicy restartPolicy = RestartPolicy.ALWAYS;
+
+
     /**
      * Applies the values of all properties of a
      * {@link MicoServiceDeploymentInfoDTO} to this
      * {@code MicoServiceDeploymentInfo}.
-     * 
+     *
      * @param serviceDeploymentInfoDTO the {@link MicoServiceDeploymentInfoDTO}.
      * @return this {@link MicoServiceDeploymentInfo} with the values
-     *         of the properties of the given {@link MicoServiceDeploymentInfoDTO}.
+     * of the properties of the given {@link MicoServiceDeploymentInfoDTO}.
      */
     public MicoServiceDeploymentInfo applyValuesFrom(MicoServiceDeploymentInfoDTO serviceDeploymentInfoDTO) {
         return setReplicas(serviceDeploymentInfoDTO.getReplicas())
-                .setMinReadySecondsBeforeMarkedAvailable(serviceDeploymentInfoDTO.getMinReadySecondsBeforeMarkedAvailable())
-                .setLabels(serviceDeploymentInfoDTO.getLabels())
-                .setImagePullPolicy(serviceDeploymentInfoDTO.getImagePullPolicy())
-                .setRestartPolicy(serviceDeploymentInfoDTO.getRestartPolicy());
+            .setMinReadySecondsBeforeMarkedAvailable(serviceDeploymentInfoDTO.getMinReadySecondsBeforeMarkedAvailable())
+            .setLabels(serviceDeploymentInfoDTO.getLabels())
+            .setImagePullPolicy(serviceDeploymentInfoDTO.getImagePullPolicy())
+            .setRestartPolicy(serviceDeploymentInfoDTO.getRestartPolicy());
     }
 
 
     /**
      * Enumeration for the different policies specifying
      * when to pull an image.
+     * Default image pull policy is {@link ImagePullPolicy#ALWAYS}.
```

Comment:
If not default values is not defined here, I think this line should be removed, same for `RestartPolicy` (and all corresponding cases).

Reply:
Currently the `MicoServiceDeploymentInfo` is used in `MicoApplication` in some endpoints. Therefore must have default values. However this should be fixed in #512